src/paas_charm/_gunicorn/charm.py

```diff
@@ -23,6 +25,60 @@ def _workload_config(self) -> WorkloadConfig:
             framework_name=self._framework_name, unit_name=self.unit.name
         )
 
+    def check_gevent_package(self) -> bool:
```

Comment:
This function and the following one could be private (they are not called anywhere else and are specific for Django). Besides, I would put it in the end of the file (see https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#function-and-method-Ordering)